src/Castle.Services.Logging.SerilogIntegration/SerilogLogger.cs

```diff
@@ -14,356 +14,419 @@
 
 namespace Castle.Services.Logging.SerilogIntegration
 {
-    using System;
+	using System;
 
-    using Serilog;
-    using Serilog.Events;
+	using Serilog;
+	using Serilog.Events;
 
 #if FEATURE_SERIALIZATION
-    [Serializable]
+	[Serializable]
 #endif
-    public class SerilogLogger :
+	public class SerilogLogger :
```

Comment:
You'll need to take another look at `SerilogLogger.cs` because it still shows as all changed.

Reply:
Reverted